implementation/.github/workflows/publish-releases.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Draft Releases
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 5 * * WED'  # Weekly on Wednesday at 5:00 AM UTC
+
+concurrency:
+  group: publish-release
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Publish Top Draft Release
```

Comment:
When we say: "top" do we mean: "top according to github", "highest semver", "chronologically most recent", or something else?

We could either rename this job or we could add a comment

Reply:
It should be "latest"

Reply:
or chron. most recent

Reply:
sorry, highest semver

Reply:
fixed